### DISCUSSION TITLE
Créer un embed pour pouvoir afficher son résultat sur son propre site internet

### DISCUSSION THREAD

**Discussion by LouisLecouturier** [cnumr/EcoIndex#270] [Ideas]
Ça pourrait-être sympa de créer une sorte de badge comme [website carbon](https://www.websitecarbon.com/badge/)
Les entreprises pourraient afficher leur scores sur leur site internet !

**Comment by vvatelot** [comment]
Bonjour @LouisLecouturier , merci pour ton message.
Un tel badge existe déjà : https://github.com/cnumr/ecoindex_badge N'hésite pas si tu as besoin de plus d'infos pour le mettre en place...
